ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -344,6 +344,26 @@ StatusSectionLayout {
                         mintPanel.errorText = qsTr("Unknown error")
                         mintPanel.isFeeLoading = true
                     }
+                    function onDeploymentStateChanged(status, url) {
+                        let title
+                        let loading = false
+                        let type = Constants.ephemeralNotificationType.normal
+                        if (status === Constants.DeployState.InProgress) {
+                            title = qsTr("Token is being minted...")
+                            loading = true
+                        } else if(status === Constants.DeployState.Deployed) {
+                            title = qsTr("Token minting finished")
+                            type = Constants.ephemeralNotificationType.success
+                        } else if (status === Constants.DeployState.Failed) {
+                            title = qsTr("Token minting failed")
+                        }
+                        Global.displayToastMessage(title,
+                                                   qsTr("View on etherscan"),
```

Comment:
Should this be a link to `etherscan` web page? When tried, I didn't show it as a link

Reply:
Because you used fake transaction hash. The url is build like this: {etherscan link}/tx/{transaction hash}